src/darwin/Framework/CHIP/CHIPDeviceController.h

```diff
@@ -102,6 +102,11 @@ typedef void (^CHIPDeviceConnectionCallback)(CHIPDevice * _Nullable device, NSEr
  */
 - (void)shutdown;
 
+/**
```

Comment:
Could you please move this to MTRCertificates? It also looks like this should be a class method (or a simple C Function), instead of an object - given this is just a utility function.